packages/ui/src/assets/icons/ArrowUp.svg

```diff
@@ -1,3 +1,3 @@
 <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M12 5L6 11M12 5L18 11M12 5V19" stroke="#1A1818" style="stroke:#1A1818;stroke:color(display-p3 0.1020 0.0941 0.0941);stroke-opacity:1;" stroke-width="1.6" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
Let's revert this for now -- `.svgrrc` is supposed to key on that color value to automatically replace color; if that's not working we should fix there.

Reply:
Yeah, doesn't appear to be working. I'll revert.

Reply:
Actually, it is working. Idk why I thought it wasn't earlier. But yeah, I'll revert.